web/api/backend.go

```diff
@@ -0,0 +1,145 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	jwtmiddleware "github.com/auth0/go-jwt-middleware"
+	"github.com/dgrijalva/jwt-go"
+
+	"github.com/gorilla/mux"
+)
+
+// Claims object that will be encoded to a JWT.
+// We add jwt.StandardClaims as an embedded type, to provide fields like expiry time
+type Claims struct {
+	UploadCredentials map[string]interface{}
+	jwt.StandardClaims
+}
+
+var jwtKey = []byte(os.Getenv("SECRET_KEY"))
+var creds WrapperCommand
+
+// TokenHandler to create the Token
```

Comment:
TokenHandler is too generic of a name, and also ambiguous in the context of oauth2l access tokens. I recommend something like "CredentialsHandler" instead. Also provide an explanation of what this function does in the comment. Discuss input and output and how the output will be used.

Reply:
Fixed!